src/pages/SearchResults/index.tsx

```diff
@@ -1,28 +1,136 @@
-import { useSearchParams, useNavigate } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
 import SearchBar from '@/components/layouts/SearchBar';
 import CategoryTabBar from './components/CategoryTabBar';
 import SwiperFrame from './components/SwiperFrame';
 import searchWork from '@/apis/data/searchWork';
 import searchArtist from '@/apis/data/searchArtist';
 import { RouterPath } from '@/routes/path';
+import { useState } from 'react';
+import ArtWorkContents from './components/ArtWorkContents';
+import ArtistContents from './components/ArtistContents';
+import styled from '@emotion/styled';
+import Gap from '@/components/styles/Gap';
+import MoreButton from './components/MoreButton';
 
 const SearchResults = () => {
-  const [searchParams] = useSearchParams();
-  const query = searchParams.get('query');
+  const [selectedTab, setSelectedTab] = useState('전체');
   const navigate = useNavigate();
+  const searchLen = searchWork.length + searchArtist.length;
+  const searchWorkLen = searchWork.length;
+  const searchArtistLen = searchArtist.length;
 
   const goBack = () => {
     navigate(RouterPath.categories);
```

Comment:
```suggestion
    navigate(-1);
```
하면 안 되나요? (다른 페이지가 중간에 껴있는지? 로직을 몰라서)

Reply:
제가 저번에 navigate(-1) 이렇게 했었는데 
멘토님이 주소로 접근이 가능할 수도 있다 하셔서 로직상 뒤로 버튼을 누르면 카테고리로 간다고 생각하여 이렇게 하였습니당